test-clients/src/main/java/com/hedera/services/bdd/suites/file/FileUpdateSuite.java

```diff
@@ -772,6 +777,142 @@ private HapiApiSpec messageSubmissionSizeChange() {
+        final ByteString bNewEvmAddress =
+                ByteString.copyFrom(CommonUtils.unhex(TxnUtils.randomHex(EVM_ADDRESS_SIZE)));
+        final ByteString cNewEvmAddress =
+                ByteString.copyFrom(CommonUtils.unhex(TxnUtils.randomHex(EVM_ADDRESS_SIZE)));
+        final AtomicReference<TokenID> fungibleTokenId = new AtomicReference<>();
+        final AtomicReference<TokenID> nonFungibleTokenId = new AtomicReference<>();
+        final AtomicReference<AccountID> aCreatedId = new AtomicReference<>();
+
+        // Note all auto-creations use the GENESIS superuser as payer
+        // account to avoid being throttled when running in CI
+        return defaultHapiSpec("HollowCreationWorksWithBothUniqueAndRepeatedUnknownEvmAddresses")
+                .given(
+                        overriding(LAZY_CREATION_PROP, "true"),
+                        tokenCreate(ft)
+                                .treasury(GENESIS)
+                                .payingWith(GENESIS)
+                                .initialSupply(1_000_000)
+                                .exposingCreatedIdTo(id -> fungibleTokenId.set(asToken(id))),
+                        tokenCreate(nft)
+                                .treasury(GENESIS)
+                                .payingWith(GENESIS)
+                                .supplyKey(GENESIS)
+                                .tokenType(TokenType.NON_FUNGIBLE_UNIQUE)
+                                .initialSupply(0)
+                                .exposingCreatedIdTo(id -> nonFungibleTokenId.set(asToken(id))),
+                        mintToken(nft, List.of(ByteString.copyFromUtf8("HELLO"))))
+                .when(
+                        // Valid hollow creation via hbar transfer
+                        cryptoTransfer(
+                                        (spec, b) ->
+                                                b.getTransfersBuilder()
+                                                        .addAccountAmounts(
+                                                                aaWithNum(2, -ONE_HUNDRED_HBARS))
+                                                        .addAccountAmounts(
+                                                                aaWith(
+                                                                        aNewEvmAddress,
+                                                                        +ONE_HUNDRED_HBARS)))
+                                .signedBy(GENESIS)
+                                .via(uniqueCreation),
+                        getTxnRecord(uniqueCreation)
+                                .andAllChildRecords()
+                                .exposingChildCreationsTo(aCreatedId::set)
+                                .hasNonStakingChildRecordCount(1),
+                        getAccountInfoWithLiteralAlias(aNewEvmAddress)
+                                // QUESTION - should we expect .hasEmptyKey() here?
```

Comment:
Suggested change is made in the target branch and now `.hasEmptyKey()` should be ok.